exercises/async-activity-completion/README.md

```diff
@@ -0,0 +1,57 @@
+# Exercise 3: Asynchronous Activity Completion
+
+During this exercise, you will:
+
+- Retrieve a task token from your Activity execution
+- Throw an `ErrResultPending` to indicate that the Activity is waiting for an external completion.using
+- Use another Temporal Client to communicate the result of the asynchronous Activity back to the Workflow
+
+Make your changes to the code in the `practice` subdirectory (look for `TODO` comments that will guide you to where you should make changes to the code). If you need a hint or want to verify your changes, look at the complete version in the `solution` subdirectory.
+
+## Part A: Retrieving the Task Token
+
+1. Edit the `workflow.go` file. In the `Activity()` definition, add a call to `activity.GetInfo()` that returns an `activityInfo` object.
+2. From that object, extract `activityInfo.TaskToken`. To asynchronously complete an Activity, you need to store this token outside of this Workflow, so that you can call it from another Temporal Client. The most straightforward way to do this is to encode it to hexadecimal or base64 and log it to your terminal, so add a call like `logger.Info("Activity", "taskToken", hex.EncodeToString(taskToken))`.
+3. Save the file.
+
+## Part B: Set Your Activity to Return `ErrResultPending`
+
+1. Continue editing the same Activity definition in the `workflow.go` file. You need to add a `return` statement that returns a special kind of error, `activity.ErrResultPending`, that will Temporal that the Activity has not failed but will be completed asynchronously.
```

Comment:
"that will Temporal that the Activity has not failed...?"

Reply:
Fixed to "that will tell Temporal," thanks!